common/src/main/java/com/viaversion/viarewind/protocol/v1_9to1_8/data/WitherBossBar.java

```diff
@@ -201,7 +201,8 @@ private void addWither() {
 		entityData.add(new EntityData(0, EntityDataTypes1_8.BYTE, (byte) 0x20));
 		entityData.add(new EntityData(2, EntityDataTypes1_8.STRING, title));
 		entityData.add(new EntityData(3, EntityDataTypes1_8.BYTE, (byte) 1));
-		entityData.add(new EntityData(6, EntityDataTypes1_8.FLOAT, health * 300f));
+		entityData.add(new EntityData(6, EntityDataTypes1_8.FLOAT, Math.max(health * 300f, 1f))); // the wither will just die if the health is too low
```

Comment:
Is this maybe related to the limit in #setHealth() above? 

`
if (this.health <= 0) {
    this.health = 0.0001f;
}
`